doc/man3/BIO_f_ssl.pod

```diff
@@ -240,6 +240,14 @@ a client and also echoes the request to standard output.
      ERR_print_errors_fp(stderr);
      exit(1);
  }
+ /* Added BIO_do_accept call after first BIO_do_accept is actually bind while *
+    second BIO_do_accept is really an accept                                 */
```

Comment:
The english still does not work here (i am not a natural english speaker either).
The sentence should say that the first do_accept made the socket bind and **listen**,
and in the second do_accept will do the **accept**.